src/pages/rest/tutorials/inventory/create-shipment.md

```diff
@@ -22,6 +22,10 @@ When you complete a partial or full shipment, Adobe Commerce deducts the reserve
 
 Although you can use the `POST V1/shipment` endpoint to create a shipment, the `POST /V1/order/:orderId/ship` endpoint is a better option in that it is more efficient and the payload can be simpler.
 
+<InlineAlert variant="notice" slots="text"/>
```

Comment:
```suggestion
<InlineAlert variant="info" slots="text"/>
```

Reply:
See the [theme](https://github.com/adobe/aio-theme?tab=readme-ov-file#inlinealert-block-updated-2022-06-08) README for a list of supported variants.